packages/library-base/src/commonMain/kotlin/io/realm/kotlin/schema/RealmClass.kt

```diff
@@ -42,7 +42,7 @@ public interface RealmClass {
     public val isEmbedded: Boolean
 
     /**
-     * Index operator to lookup a specific [RealmProperty] from it's property name.
+     * Index operator to lookup a specific [RealmProperty] from it's internal property name.
```

Comment:
```suggestion
     * Index operator to lookup a specific [RealmProperty] from it's persisted property name.
```

`internal` feels a bit too, well internal, given our discussion, maybe persisted is as better way to phrase it